training/deepspeech_training/util/audio.py

```diff
@@ -21,8 +23,9 @@
 AUDIO_TYPE_PCM = 'application/vnd.mozilla.pcm'
 AUDIO_TYPE_WAV = 'audio/wav'
 AUDIO_TYPE_OPUS = 'application/vnd.mozilla.opus'
-SERIALIZABLE_AUDIO_TYPES = [AUDIO_TYPE_WAV, AUDIO_TYPE_OPUS]
-LOADABLE_AUDIO_EXTENSIONS = {'.wav': AUDIO_TYPE_WAV}
+AUDIO_TYPE_OGG_OPUS = 'application/vnd.deepspeech.ogg_opus'
```

Comment:
why do we need this?

Reply:
MLS uses `.opus` files which are specified as `audio/ogg`, which is the same for every Ogg container. But our code only actually handles Ogg with Opus encoded audio inside, which is not `audio/ogg`. In any case it's not like these mime types are an exposed API, I just didn't want to take `audio/ogg` in case one day we move towards supporting proper Ogg containers with more codecs.